strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java

```diff
@@ -89,6 +90,10 @@ protected void configure(HttpSecurity http)
             // TODO SB-813
             .authenticationEntryPoint(customBasicAuthenticationEntryPoint())
             .and()
+            .authorizeRequests()
+            .requestMatchers(EndpointRequest.toAnyEndpoint())
+            .hasAuthority("ADMIN")
```

Comment:
@ptirador shouldn't these be set under `application.properties` as `management.security.role=ADMIN` ?

Reply:
Yes, totally.